src/main/java/com/app/repositories/user/UserRepository.java

```diff
@@ -15,4 +21,14 @@
     Boolean existsByUsername(String username);
     Boolean existsByEmail(String email);
 
-}
+    @Query("select u.verified from User u where username = :username")
```

Comment:
To trochę mało Springowe rozwiązanie. Query jest zupełnie niepotrzebne, Hibernate sam resolvuje po nazwach metody jakie ma być query. Lepiej po prostu pobrać danego usera do servisu i tam go bodyfikować, czyli metody findByUserane i findByEmail, ustawianie wtedy w serivice

Reply:
Generalnie chodzi o to też żeby oddzielić warstwę prezentacji od warstwy logicznej. U nas teraz controller korzysta bezpośrednio z repository, na szybko było ok ale generalnie jest to błędem. Logika tworzenia użytkownika, logowania i tej weryfikacji powinna być wydzielona do specjalnego UserService.